electrumx/lib/util_atomicals.py

```diff
@@ -1664,6 +1663,21 @@ def get_subname_request_candidate_status(current_height, atomical_info, status,
         'pending_candidate_atomical_id': candidate_id_compact
     }
 
+# Whether txid is valid for the current and next bitwork
+def is_txid_valid_for_bitwork(txid, bitwork_vec, actual_mints, max_mints, target_increment, starting_target, allow_next):
```

Comment:
For readability: rename to `is_txid_valid_for_perpetual_bitwork`

Reply:
Done